common/customizable_schema.go

```diff
@@ -149,31 +219,47 @@ func (s *CustomizableSchema) SetConflictsWith(value []string) *CustomizableSchem
 	if len(value) == 0 {
 		panic("SetConflictsWith cannot take in an empty list")
 	}
-	s.Schema.ConflictsWith = value
+	if s.pathContainsMultipleItemsList() {
```

Comment:
Fine to return a warning in this case. However, if `path` is `nil` (i.e. this is for a top-level resource), we should panic if this happens, because it will never work as expected.

This applies to all cases.